services.html

```diff
@@ -0,0 +1,148 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <title>Great Idea! - Services</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+  <body>
+
+    <div class="container">
+
+      <nav>
+
+        <a href="services.html">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+
+        <a href="index.html" class="logo"><img src="img/logo.png" alt="Great Idea! Company logo."></a>
```

Comment:
I think I commented about this on yesterday's project, but I'd probably spread this out to three lines for readability:
```html
<a href="index.html" class="logo">
  <img src="img/logo.png" alt="Great Idea! Company logo.">
</a>
```